server/tests/integration/process/email.test.js

```diff
@@ -336,5 +379,39 @@ describe('Emails', function() {
       (seventhCallTo === secondUserEmail && eighthCallTo === firstUserEmail),
 			'sendMail should have been called to send to the user\'s email'
 		);
+
+		planPersonModel.subscribe (secondPerson.id, firstIPlanDb.id);
+		planPersonModel.subscribe (secondPerson.id, secondIPlanDb.id);
+	
+		const newMavatScope = nock('https://mavat.iplan.gov.il', { allowUnmocked: true })
+		.persist()
+		.get('/rest/api/SV4/1/?mid=456456')
+		// actual reply copied from a browser performing the API response
+		.replyWithFile(
+			200,
+			`${__dirname}/files/new_mavat_plan_json_page.html`,
+			{ 'Content-Type': 'text/html' }
+		);
+
+		const newMavatScopeNotDone = nock('https://mavat.iplan.gov.il', { allowUnmocked: true })
+		.get('/rest/api/SV4/1/?mid=123123')
+		// actual reply copied from a browser performing the API response
+		.replyWithFile(
+			200,
+			`${__dirname}/files/new_mavat_plan_json_page_start.html`,
+			{ 'Content-Type': 'text/html' }
+		);
+
+		await cronController.fetchPlanStatus();
+
+		newMavatScope.done();
```

Comment:
IMO let's separate this into two different tests:
The first is testing the new plan email
The second will test the status email send.

The same logic can be copied